src/components/TodoInput.js

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { HiPlusCircle } from 'react-icons/hi';
+import { addTodo } from '../features/todo';
+
+const TodoInput = () => {
+  const [input, setInput] = useState('');
+  const dispatch = useDispatch();
+
+  const handleSubmit = () => {
```

Comment:
As the event, you are handling is a click, the event handler name should reflect that.